src/foam/nanos/auth/LastModifiedAwareDAO.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.auth;
```

Comment:
Are these classes really nanos specific? Or would they do better in a more generate purpose package?